spring-cloud-gcp-bigquery/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>spring-cloud-gcp</artifactId>
+    <groupId>org.springframework.cloud</groupId>
+    <version>1.2.0.BUILD-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>spring-cloud-gcp-bigquery</artifactId>
+  <name>Spring Cloud GCP BigQuery</name>
+  <description>Spring Cloud GCP BigQuery Module</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.integration</groupId>
+      <artifactId>spring-integration-core</artifactId>
```

Comment:
Why have you resolved this conversation, since I don't see `<optional>true</optional>` in this dependency?
Although it looks like you have placed it accidentally at the  `google-cloud-bigquery`.
For me logically must be opposite...

Reply:
oops, thanks for your Vigilance